routes/api/status/index.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = async function(fastify, opts) {
+  fastify.get('/', async (request, reply) => {
+    const kubeContext = fastify.kube.currentContext;
+    let body = {};
+    const {currentContext, namespace} = fastify.kube;
+    if ( !kubeContext && !kubeContext.trim()) {
+      body.status = 'Error';
+      body.message = 'Unable to connect to Kube API';
+      reply.code(500);
+    } else {
+      body.status = 'OK'
+      body.kube = {
+        currentContext,
+        namespace
+      };
+      reply.code(200);
+    }
+    reply.send(body);
+    return reply;
+  });
+};
```

Comment:
missing new line

Reply:
I'll add a `prettier` script to `package.json` so files don't get missed.